src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectFile.cs

```diff
@@ -54,12 +54,12 @@ public string GetPropertyValue(string name)
 
         protected class BuildResult
         {
-            public readonly MSB.Execution.BuildResult Result;
+            public readonly bool Built;
```

Comment:
OK, will do. Should I get rid of the `BuildResult` class as well then, since it would hold just the MSBuild executed project's `ProjectInstance`?